src/controllers/index.js

```diff
@@ -4,10 +4,12 @@ const signup = require('./signup');
 const specialize = require('./specialization');
 const login = require('./login');
 const error = require('./error');
+const authorization = require('../middlewares/authorization');
 
 const router = express.Router();
 
 router.get('/login', login.get);
+router.use(authorization);
 router.post('/specialize', specialize);
 router.route('/signup')
```

Comment:
re-oreder you routes and put login and sign up before ``` auth ``` middleware

Reply:
@knanahassouna1  done